My FTB server isnt working because of forestry

My server all of a sudden stopped working and came up with this error, i think this is where it starts:

[20:12:43] [Server thread/ERROR] [minecraft/MinecraftServer]: Encountered an unexpected exception
net.minecraft.util.ReportedException: Ticking entity
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:773) ~[MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) ~[nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) ~[MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_221]
Caused by: java.lang.IllegalArgumentException: bound must be positive
        at java.util.Random.nextInt(Unknown Source) ~[?:1.8.0_221]
        at forestry.arboriculture.blocks.BlockAsh.getDrops(BlockAsh.java:83) ~[BlockAsh.class:?]
        at net.minecraft.block.Block.getDrops(Block.java:1300) ~[aow.class:?]
        at mcjty.meecreeps.actions.workers.WorkerHelper.harvestAndPickup(WorkerHelper.java:660) ~[WorkerHelper.class:?]
        at mcjty.meecreeps.actions.workers.DigTunnelActionWorker.dig(DigTunnelActionWorker.java:93) ~[DigTunnelActionWorker.class:?]
        at mcjty.meecreeps.actions.workers.DigTunnelActionWorker.lambda$null$0(DigTunnelActionWorker.java:187) ~[DigTunnelActionWorker.class:?]
        at mcjty.meecreeps.actions.workers.WorkerHelper.delayForHardBlocks(WorkerHelper.java:293) ~[WorkerHelper.class:?]
        at mcjty.meecreeps.actions.workers.DigTunnelActionWorker.lambda$tick$1(DigTunnelActionWorker.java:187) ~[DigTunnelActionWorker.class:?]
        at mcjty.meecreeps.actions.workers.WorkerHelper.navigateTo(WorkerHelper.java:401) ~[WorkerHelper.class:?]
        at mcjty.meecreeps.actions.workers.DigTunnelActionWorker.tick(DigTunnelActionWorker.java:186) ~[DigTunnelActionWorker.class:?]
        at mcjty.meecreeps.actions.workers.WorkerHelper.tick(WorkerHelper.java:508) ~[WorkerHelper.class:?]
        at mcjty.meecreeps.entities.MeeCreepWorkerTask.func_75246_d(MeeCreepWorkerTask.java:69) ~[MeeCreepWorkerTask.class:?]
        at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:129) ~[xf.class:?]
        at net.minecraft.entity.EntityLiving.func_70626_be(EntityLiving.java:763) ~[vq.class:?]
        at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2359) ~[vp.class:?]
        at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:577) ~[vq.class:?]
        at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2179) ~[vp.class:?]
        at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:295) ~[vq.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:1993) ~[amu.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832) ~[oo.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:1955) ~[amu.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:1759) ~[amu.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613) ~[oo.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767) ~[MinecraftServer.class:?]
        ... 4 more
[20:12:43] [Server thread/ERROR] [minecraft/MinecraftServer]: This crash report has been saved to: C:\Users\Owen\Desktop\Mincraft server\Direwolf Server\FTB+Presents+Direwolf20+1.12-1.12.2-2.5.0-Server\.\crash-reports\crash-2019-07-30_20.12.43-server.txt
[20:12:43] [Server thread/INFO] [minecraft/MinecraftServer]: Stopping server
[20:12:43] [Server thread/INFO] [minecraft/MinecraftServer]: Saving players
[20:12:43] [Server thread/INFO] [minecraft/MinecraftServer]: Saving worlds
[20:12:43] [Server thread/INFO] [minecraft/MinecraftServer]: Saving chunks for level 'world'/overworld
[20:12:43] [Thread-25/INFO] [THAUMCRAFT]: Stopping aura thread for dim 1
[20:12:43] [Thread-26/INFO] [THAUMCRAFT]: Stopping aura thread for dim -1
[20:12:43] [Thread-27/INFO] [THAUMCRAFT]: Stopping aura thread for dim 144
[20:12:43] [Thread-28/INFO] [THAUMCRAFT]: Stopping aura thread for dim 111
[20:12:43] [Server thread/INFO] [minecraft/MinecraftServer]: Saving chunks for level 'world'/ExtraUtils2_Quarry_Dim
[20:12:43] [Thread-30/INFO] [THAUMCRAFT]: Stopping aura thread for dim -11325
[20:12:43] [Thread-29/INFO] [THAUMCRAFT]: Stopping aura thread for dim -6
[20:12:43] [Server thread/INFO] [minecraft/MinecraftServer]: Saving chunks for level 'world'/Storage Cell
[20:12:43] [Server thread/INFO] [THAUMCRAFT]: Removing aura cache for world 0
[20:12:43] [Thread-33/INFO] [THAUMCRAFT]: Stopping aura thread for dim 7
[20:12:43] [Server thread/INFO] [THAUMCRAFT]: Removing aura cache for world -9999
[20:12:43] [Server thread/INFO] [THAUMCRAFT]: Removing aura cache for world 2
[20:12:43] [Server thread/INFO] [FML]: Unloading dimension 0
[20:12:43] [Server thread/INFO] [FML]: Unloading dimension -9999
[20:12:43] [Server thread/INFO] [FML]: Unloading dimension 2
[20:12:43] [Server thread/INFO] [Actually Additions]: Unloaded WorldData!
[20:12:43] [Server thread/INFO] [mcjty.lib.varia.Logging]: Cleaning up all world data: 4 data blobs
[20:12:43] [Server thread/INFO] [mcjty.lib.varia.Logging]: RFTools Dimensions: server is stopping. Shutting down gracefully
[20:12:43] [Server thread/INFO] [mcjty.lib.varia.Logging]: Cleaning up RFTools dimensions
[20:12:43] [Server thread/INFO] [FML]: The state engine was in incorrect state SERVER_STOPPING and forced into state SERVER_STOPPED. Errors may have been discarded.
[20:12:43] [Server Shutdown Thread/INFO] [minecraft/MinecraftServer]: Stopping server
[20:12:43] [Server Shutdown Thread/INFO] [minecraft/MinecraftServer]: Saving players
[20:12:43] [Server Shutdown Thread/INFO] [minecraft/MinecraftServer]: Saving worlds

I have attached the crash report as well
[crash-2019-07-30_20.12.43-server.txt](https://github.com/ForestryMC/ForestryMC/files/3448722/crash-2019-07-30_20.12.43-server.txt)

--- a/src/main/java/forestry/arboriculture/blocks/BlockAsh.java
+++ b/src/main/java/forestry/arboriculture/blocks/BlockAsh.java
@@ -80,7 +80,7 @@ public class BlockAsh extends Block implements IStateMapperRegister, IItemModelR
 				amount += rand.nextInt(1 + fortune);
 			}
 			drops.add(new ItemStack(Items.COAL, amount, 1));
-			drops.add(new ItemStack(ModuleCore.getItems().ash, 1 + rand.nextInt(amount / 4)));
+			drops.add(new ItemStack(ModuleCore.getItems().ash, 1 + rand.nextInt(Math.max(amount / 4, 1))));
 		}
 	}
 

